### ISSUE TITLE
Suggestion.

### ISSUE THREAD

**Issue by LeeRyanSA** [ADawesomeguy/AwesomeSciBo#39]
If I would suggest, you should make a function in the bot that will compile a bunch of problems together into one pdf, based on the preferences of the user. For example, number of problems, distribution of problems, problem level, or other things. Would be really helpful for practice competitions, and allows for one to practice problems daily without looking at a discord screen every day.

**Comment by xlq902** [comment]
Already exists in part with /rounds, but that needs to be fixed with #30 .

**Comment by ADawesomeguy** [comment]
Yes, this exactly. It will ideally also tie into #35. Honestly, the bot was originally made solely for distributing custom-made rounds if you take a look at the [initial tag](https://github.com/ADawesomeguy/AwesomeSciBo/tree/v1.0.0).

**Comment by ADawesomeguy** [comment]
Ok for now it should be fixed in terms of hosting, but I'll close this issue with #30.
